lib/component.js

```diff
@@ -125,6 +128,14 @@ export default class Component extends WebComponent {
     return true;
   }
 
+  toString() {
+    try {
+      return `${this.tagName}#${this.panelID}`;
```

Comment:
In the current implementation, none within reason. But if for example `this.panelID` became a getter and executed code, it could potentially throw. `toString()` should be safe to call under basically any circumstances, especially considering that it's used for error output in a `catch` clause below. Few things more frustrating than getting an error which you can't even see because your error handler blew up.